webviz_ert/controllers/observation_response_controller.py

```diff
@@ -113,8 +113,8 @@ def update_graph(
                     selection=None
                 )  # What about selections?
                 if summary_df is not None:
-                    data_dict[parent.ensembles[ensemble_id]._name] = summary_df
-                colors[parent.ensembles[ensemble_id]._name] = color["color"]
+                    data_dict[ensemble._name] = summary_df
```

Comment:
Should we do a refactoring / expose name as a property in EnsembleModel class avoid the use of private members?

Reply:
Good idea, I avoided doing it as part of this PR given that it was not really related to what I was trying to do. But I don't mind doing it as part of this PR if you are also OK with it.

Reply:
Done